ci/ray_ci/tester_container.py

```diff
@@ -15,6 +15,7 @@ class TesterContainer(Container):
     def __init__(
         self,
         docker_tag: str,
+        operating_system: str = "linux",
```

Comment:
please use `Optional[str] = None`, and if it is None, set it to "linux".

Otherwise, this `"linux"` will be initialized as a global valuable, and if its content is mutated in the function, it will have unintended mysterious effects.